react/features/base/settings/reducer.js

```diff
@@ -0,0 +1,162 @@
+// @flow
+
+import { APP_WILL_MOUNT } from '../../app';
+import JitsiMeetJS, { browser } from '../lib-jitsi-meet';
+import { ReducerRegistry } from '../redux';
+import { PersistenceRegistry } from '../storage';
+import { randomHexString } from '../util';
+
+import { SETTINGS_UPDATED } from './actionTypes';
+
+const logger = require('jitsi-meet-logger').getLogger(__filename);
+
+/**
+ * The default/initial redux state of the feature {@code base/settings}.
+ *
+ * @type Object
+ */
+const DEFAULT_STATE = {};
```

Comment:
we should have some default state

Reply:
No, config.js could have many undefined values, we should have some sane defaults ourselves.